ui/app/components/dashboard/secrets-engines-card.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class DashboardSecretsEnginesCardComponent extends Component {
```

Comment:
NIT: you don't need to end the name of the component with "Component". Similar to `KeymgmtDistribute` component, it's named the file path without the end of the component. Curious, did this naming come default from the blueprints? So name would be `DashboardSecretsEnginesCard`

Reply:
Though I do see at least one use case where we do end in component. PolicyFormComponent. So ignore, I think it's a personal preference. Doesn't look like a hard fast rule.

Reply:
I do see the redundancy - I will remove it! I think we added Component to the end in the PKI and the blueprint automatically creates the name with the Component in the end.